### PR TITLE
Removing inline CSS

### PR DESCRIPTION
**Pull request by ADFF2F**
Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [x] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [x] Any new functionality is covered by tests

### CODE REVIEW

File: themes/pmahomme/css/common.css.php [hunk 1]
```diff
@@ -2503,6 +2501,13 @@
     padding: 2px;
 }
 
+.clearright {
+    clear:right;
+}
```
Comment (nijel) [inline themes/pmahomme/css/common.css.php]: This is most likely wrong for RTL languages (probably should match floatright).

File: themes/pmahomme/css/common.css.php [hunk 2]
```diff
@@ -2503,6 +2501,13 @@
...
+.displaynone {
+    display:none;
+}
```
Comment (nijel) [inline themes/pmahomme/css/common.css.php]: There is already hide class doing this, no need to duplicate it.

**Review by nijel — CHANGES_REQUESTED** [review]
Thanks for your patch. Besides things I've mentioned inline, the CSS changes should be also implemented in second theme we ship - in themes/original/css/common.css.php

File: themes/pmahomme/css/common.css.php (hunk 1 above)
Comment (skokeers27) [reply themes/pmahomme/css/common.css.php]: This is original line:
echo '<div class="choice floatright" style="clear:right;">';
Shoudl I left 2 classes, choice and floatright and remove "style="clear:right;" ?

Comment (nijel) [reply themes/pmahomme/css/common.css.php]: Yes, the original code was wrong as well. Hmm, looking at it, this element seems to be hidden all the time anyway, so it's probably better remove anything else from it:

https://github.com/phpmyadmin/phpmyadmin/blob/701a8855fdb4c992e3882b76c40a92c2610891ba/themes/pmahomme/css/gis.css.php#L46-L48

Comment (skokeers27) [reply themes/pmahomme/css/common.css.php]: Ok, I left only choice class.

**Review by ADFF2F — COMMENTED** [review]
changes have been made

**Comment by nijel** [comment]
Thanks, I've merged this PR with following fixes:

* 8109763b75 Remove not needed CSS for span elements
* 43cbcf6bd0 Remove unused class
* 636b22cb06 Remove clearright class

**Pull request merged**